versions.gradle

```diff
@@ -11,14 +11,14 @@ ext {
             archRoom                       : '1.1.1',
             archTesting                    : '1.1.1',
             dagger                         : '2.15',
-            detekt                         : '1.0.0.RC7-3',
+            detekt                         : '1.0.0.RC9.2',
```

Comment:
is `.` ok? the previous version had a `-` and not a `.` that's why I'm asking

Reply:
I suppose they've changed it

Reply:
yep, they changed it